ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRScanViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Windows.Input;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+using ZXing;
+
+namespace ScanShop.Mobile.ViewModels
+{
+    public class QRScanViewModel : BaseViewModel
+    {
+        public Command ScanCommand { get; set; }
```

Comment:
It would be better to add newlines between fields, methods, constructors etc.

Reply:
Fixed